src/ui-test/camelk_didact_tutorial_test.ts

```diff
@@ -0,0 +1,124 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { expect } from 'chai';
+import {
+	ActivityBar,
+	By,
+	EditorView,
+	until,
+	ViewItem,
+	ViewSection,
+	WebView
+} from 'vscode-extension-tester';
+
+describe('Didact Tutorial of Camel K', function () {
+
+	let switchedContextWebview: WebView | undefined;
+	let handle: string | undefined;
```

Comment:
this is used only for the workaround https://github.com/camel-tooling/vscode-camelk/pull/803/files#diff-18e1136d2c6b95271938ca7cb0318c33eed3f81fd6986c6be12d0499872a536fR83